tasks/cephfs/test_volume_client.py

```diff
@@ -23,11 +23,11 @@ def _volume_client_python(self, client, script):
 log = logging.getLogger("ceph_volume_client")
 log.addHandler(logging.StreamHandler())
 log.setLevel(logging.DEBUG)
-vc = CephFSVolumeClient("manila", "{conf_path}", "ceph")
+vc = CephFSVolumeClient("manila", "{conf_path}", "ceph", {vol_prefix}, {ns_prefix})
```

Comment:
this is in format, so if vol_prefix = None, it will be format like "None" if "None" else None, which is also wrong.

Change line 115 can works, but will make the function very strange...

Is that better to do the quotes escape  in _volume_client_python like 
if vol_prefix:
    vol_prefix = "\"" + vol_prefix + "\""